docs/how-to-use-gvfs.md

```diff
@@ -0,0 +1,145 @@
+slug: /how-to-use-gvfs
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Introduction
+
+Gravitino supports operating the storage mapping to `fileset` through the Hadoop Compatible File System (HCFS) interface,
+which names `Gravitino Virtual File System` (It is called `gvfs` for short).  
+You can access the storage mapping to fileset through the following path:
+```text
+gvfs://fileset/${catalog_name}/${schema_name}/${fileset_name}/sub_dir/
+```
+The following mainly introduces how to use `Gravitino Virtual File System` to operate fileset which mapping to HDFS.
+
+
+## Prerequisites
+
++ Hadoop environment (Hadoop 3.1.0 has been tested)
++ HDFS (Best matched to Hadoop environment)
++ Already created a fileset mapping to HDFS through the Gravitino server
+
+
+## Mainly Configuration
+
+| Configuration item                                    | Description                                                                                                                                                                                       | Default value | Required | Since version |
+|-------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `fs.AbstractFileSystem.gvfs.impl`                     | The Gravitino Virtual File System abstract class. Please configure it as `com.datastrato.gravitino.filesystem.hadoop3.Gvfs`.                                                                      | (none)        | Yes      | 0.5.0         |
+| `fs.gvfs.impl`                                        | The Gravitino Virtual File System implementation class. Please configure it as `com.datastrato.gravitino.filesystem.hadoop3.GravitinoVirtualFileSystem`.                                          | (none)        | Yes      | 0.5.0         |
+| `fs.gvfs.impl.disable.cache`                          | Close the Gravitino Virtual File System cache in Hadoop environment. If you need to proxy multi-user operations, please set this value to `true` and create a separate File System for each user. | `false`       | No       | 0.5.0         |
+| `fs.gravitino.server.uri`                             | The Gravitino server uri which gvfs needs to load the fileset meta.                                                                                                                               | (none)        | Yes      | 0.5.0         |
+| `fs.gravitino.client.metalake`                        | The metalake which fileset belongs.                                                                                                                                                               | (none)        | Yes      | 0.5.0         |
```

Comment:
I'm not sure `fs.gravitino.client.metalake` is a config option name.

Reply:
Currently will use this config to initialize the Gravitino Client:
```
GravitinoClient.builder(serverUri).withMetalake(metalakeName).withSimpleAuth().build();
```

Reply:
> I'm not sure `fs.gravitino.client.metalake` is a config option name.

Sorry, I mean that `fs.gravitino.client.metalake` is a good config option.

Reply:
Think twice. It may be ok. I mark this comment resolved.